golem_messages/validators.py

```diff
@@ -24,14 +24,24 @@ def validate_varchar(field_name, value, max_length):
 
 
 def validate_integer(field_name, value):
-    if not isinstance(value, int):
+    if isinstance(value, bool) or not isinstance(value, int):
```

Comment:
?

Reply:
remove it and unit test will break, check it out yourself

Reply:
Didn't know that bools are valid python ints.
```python
>>> isinstance(True, int)
True
>>> True + 1
2
>>> False + 1
1
>>> True + False
1
```

Reply:
yup, me too, till y-day